src/main/java/codeu/model/data/Tag.java

```diff
@@ -27,6 +30,11 @@ public UUID getId() {
     return id;
   }
 
+  /** Returns the ID of the Conversation this Tag belongs to. */
+  public UUID getConversationId() {
+    return conversation;
+  }
+
   /** Returns the content of this Tag. */
   public String getTag() {
     return tag;
```

Comment:
can you change the title of this method? to reflect that it gets the content of the tag and not the object itself